chore/ci-extra.sh

```diff
@@ -93,13 +93,3 @@ git diff
 mvn -q -Djava.src.version=11 test
 mvn -q checkstyle:checkstyle license:check
 mvn -q depclean:depclean
-
-##################################################################
-## Trigerring extra tasks that we don't want to commit to master
-## (For experimental CI features, short lived tasks, etc)
-
-if [[ "$TRAVIS_REPO_SLUG" == "INRIA/spoon" ]] && [[ "$TRAVIS_PULL_REQUEST" != "false" ]]
-then
-  echo "downloading extra CI PR script from SpoonLabs/spoon-ci-external"
-  curl https://raw.githubusercontent.com/SpoonLabs/spoon-ci-external/master/spoon-pull-request.sh | bash
```

Comment:
we may keep that, it's cool for experimenting. How to update the if condition in GHA?

Reply:
Re-added, you can see here that the "downloading extra CI PR..." is now being executed: https://github.com/INRIA/spoon/runs/2387878660#step:8:3814